docs/app/Examples/modules/Popup/Variations/PopupWidthExample.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Icon, Popup } from 'stardust'
+
+export default function () {
+  return (
+    <div>
+      <Popup
+        trigger={<Icon circular name='heart' />}
+        content='Hello. This is a wide pop-up which allows for lots of content
```

Comment:
Single quotes can't handle line breaks.  Though we could use backticks to preserve the line breaks, I think we should instead use `children` here:

```jsx
<Popup trigger={<Icon circular name='heart' />} width='wide' hover>
  Hello. This is a very wide pop-up which allows for lots of content with additional space.
  You can fit a lot of words here and the paragraphs will be pretty wide.
</Popup>
```